packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -118,7 +117,7 @@ export class FluidDataStoreRuntime
 			context,
 			sharedObjectRegistry,
 			existing,
-			async (dataStoreRuntime) => requestFluidObject(dataStoreRuntime, "/"),
+			async (dataStoreRuntime) => dataStoreRuntime.entryPoint,
```

Comment:
This change was missed in https://github.com/microsoft/FluidFramework/pull/17040, and I'd rather stick it in this PR than a separate one since velocity in next is slower than main.